firestore-bigquery-export/scripts/import/src/config.ts

```diff
@@ -147,30 +147,48 @@ const questions = [
   },
 ];
 
-export async function parseConfig(): Promise<CliConfig> {
+export async function parseConfig(): Promise<CliConfig | CliConfigError> {
```

Comment:
Hi @dackers86,

This is also not a change made in this PR, also, this function was implemented twice as the above signatures, I kept the second one since it is reusable.

Reply:
Ok, that's fine! Resolving.